gdal/frmts/daas/daasdataset.cpp

```diff
@@ -2222,6 +2222,10 @@ CPLErr GDALDAASRasterBand::GetBlocks(int nBlockXOff, int nBlockYOff,
     {
         oTargetModel.Add("sampling-algo", "AVERAGE");
     }
+    else if( poGDS->m_eCurrentResampleAlg == GRIORA_RMS )
+    {
+        oTargetModel.Add("sampling-algo", "RMS");
+    }
```

Comment:
@rouault 
I'm not shure if it possible while I don't know if it exist in DAAS!
should I remove this value from DAAS driver?

Reply:
> should I remove this value from DAAS driver?

yes, please revert that change. I'm pretty sure the DAAS API doesn't support this